csrc/mmdeploy/apis/java/mmdeploy/PoseTracker.java

```diff
@@ -0,0 +1,129 @@
+package mmdeploy;
+
+public class PoseTracker {
+    static {
+        System.loadLibrary("mmdeploy_java");
+    }
+
+    private final long handle;
+    private long stateHandle;
+
+    public static class Result {
+        public PointF[] keypoints;
+        public float[] scores;
+        public Rect bbox;
+        public int targetID;
+        public Result(PointF[] keypoints, float[] scores, Rect bbox, int targetID) {
+            this.keypoints = keypoints;
+            this.scores = scores;
+            this.bbox = bbox;
+            this.targetID = targetID;
+        }
+    }
+
+    public static class Params {
+        public int detInterval;
+        public int detLabel;
+        public float detThr;
+        public float detMinBboxSize;
+        public float detNmsThr;
+        public int poseMaxNumBboxes;
+        public float poseKptThr;
+        public int poseMinKeypoints;
+        public float poseBboxScale;
+        public float poseMinBboxSize;
+        public float poseNmsThr;
+        public float[] keypointSigmas;
+        public int keypointSigmasSize;
+        public float trackIouThr;
+        public int trackMaxMissing;
+        public int trackHistorySize;
+        public float stdWeightPosition;
+        public float stdWeightVelocity;
+        public float[] smoothParams;
+        public Params(int detInterval, int detLabel, float detThr, float detMinBboxSize, float detNmsThr, int poseMaxNumBboxes,
+                    float poseKptThr, int poseMinKeypoints, float poseBboxScale, float poseMinBboxSize, float poseNmsThr, float[] keypointSigmas,
+                    int keypointSigmasSize, float trackIouThr, int trackMaxMissing, int trackHistorySize, float stdWeightPosition, float stdWeightVelocity,
+                    float[] smoothParams) {
+                        this.detInterval = detInterval;
+                        this.detLabel = detLabel;
+                        this.detThr = detThr;
+                        this.detMinBboxSize = detMinBboxSize;
+                        this.detNmsThr = detNmsThr;
+                        this.poseMaxNumBboxes = poseMaxNumBboxes;
+                        this.poseKptThr = poseKptThr;
+                        this.poseMinKeypoints = poseMinKeypoints;
+                        this.poseBboxScale = poseBboxScale;
+                        this.poseMinBboxSize = poseMinBboxSize;
+                        this.poseNmsThr = poseNmsThr;
+                        this.keypointSigmas = new float[keypointSigmasSize];
```

Comment:
`this.keypointSigmas = keypointSigmas.clone();`
https://stackoverflow.com/questions/5785745/make-copy-of-an-array